ibis/omniscidb/tests/conftest.py

```diff
@@ -200,3 +212,77 @@ def temp_database(con, test_data_db: str) -> str:
     finally:
         con.set_database(test_data_db)
         con.drop_database(name, force=True)
+
+
+@pytest.fixture
+def temp_view(con) -> str:
+    """Return a temporary view name.
+
+    Parameters
+    ----------
+    con : ibis.omniscidb.OmniSciDBClient
+
+    Yields
+    ------
+    name : string
+        Random view name for a temporary usage.
+    """
+    name = _random_identifier('view')
+    try:
+        yield name
+    finally:
+        con.drop_view(name, force=True)
+
+
+@pytest.fixture(scope='function')
+def test_view(con, test_table) -> ibis.expr.types.TableExpr:
+    """Create a temporary view.
```

Comment:
There is a lot of repetitive code in these functions, `temp_table`, `temp_view`, `test_view`...

Do you think would make sense to create a generic function to avoid the repetition? Something like:
```python
def test_helper(con, kind='table', create=False):
    ...

@pytest.fixture
def temp_view(con):
    return test_helper(con, kind='view', create=False)
```

Or probably something even better can be done, but personally I think the fixtures part starts to be too long and repetitive.

Reply:
done